cmd/teller/teller.go

```diff
@@ -219,21 +215,20 @@ func run() error {
 			scanService = btcScanner
 		}
 
-		// enable eth scanner
-		if cfg.EthRPC.Enabled {
-			ethScanner, err = createEthScanner(rusloggger, cfg, scanStore)
+		// create sky scanner if its enabled
+		if cfg.SkyRPC.Enabled {
+			skyScanner, err = createSkyScanner(rusloggger, cfg, scanStore)
 			if err != nil {
-				log.WithError(err).Error("create eth scanner failed")
+				log.WithError(err).Error("create sky scanner failed")
 				return err
 			}
 
-			background("ethScanner.Run", errC, ethScanner.Run)
+			background("skyscanner.Run", errC, skyScanner.Run)
 
-			scanEthService = ethScanner
+			scanSkyService = skyScanner
 
-			if err := multiplexer.AddScanner(scanEthService, scanner.CoinTypeETH); err != nil {
-				log.WithError(err).Errorf("multiplexer.AddScanner of %s failed", scanner.CoinTypeETH)
-				return err
+			if err := multiplexer.AddScanner(scanSkyService, scanner.CoinTypeSKY); err != nil {
+				log.WithError(err).Errorf("multiplexer.AddScanner of %s failed", scanner.CoinTypeSKY)
```

Comment:
Missing `return err`?

Reply:
ah yes.

Reply:
yes.

Reply:
yes. If there is error with adding a sky scanner, return that error. Otherwise, return nil (later on).